main.py

```diff
@@ -0,0 +1,49 @@
+import ssl
+from time import time
+from datetime import datetime
+from urllib.parse import urlparse
+from pathlib import Path
+import telebot
+
+# access token for telegram bot
+access_token = ''
```

Comment:
It seems to me that it would be a good idea to get the value of this variable from the environment variable